draft-ietf-tls-certificate-compression.md

```diff
@@ -64,14 +65,19 @@ meaning defined in [RFC2119].
 
 # Negotiating Certificate Compression
 
-This document defines a new extension type (compress_certificates(TBD)), which
-is used by the client and the server to negotiate the use of compression for
-their certificate chains, as well as the choice of the compression algorithm.
+This document defines a new extension type (compress_certificate(TBD)), which
+can be used to signal the supported compression formats for the Certificate
+message to the peer.  Whenever it is sent by the client as a ClientHello message
+extension ([RFC5246], Section 7.4.1.4), it indicates the support for compressed
+server certificates.  Whenever it is sent by the server as a CertificateRequest
+extension ([I-D.ietf-tls-tls13], Section 4.3.2), it indicates the support for
+compressed client certificates.  Note that CertificareRequest messages are not
+supported by TLS versions prior to 1.3.
 
-By sending the compress_certificates extension, the client indicates to the
-server the certificate compression algorithms it supports.  The "extension_data"
-field of this extension in the ClientHello SHALL contain a
-CertificateCompressionAlgorithms value:
+By sending a compress_certificate extension, the sender indicates to the peer
+the certificate compression algorithms it is willing to decompress.  The
+"extension_data" field of this extension in the Extension message SHALL contain
```

Comment:
Is the uppercase E intentional?

Reply:
It was a reference to the TLS struct named "Extension".  David was confused about this too, so I just removed the reference since I think it's clear as-is.